scripts/presubmit-tests.sh

```diff
@@ -103,21 +107,33 @@ function run_build_tests() {
   return ${failed}
 }
 
-# Default build test runner that:
-# * lint and link check markdown files
-# * `go build` on the entire repo
-# * run `/hack/verify-codegen.sh` (if it exists)
-# * check licenses in `/cmd` (if it exists)
-function default_build_test_runner() {
-  local failed=0
-  # Ignore markdown files in /vendor
+# Perform markdown build tests if necessary, unless disabled.
+function markdown_build_tests() {
+  (( DISABLE_MD_LINTING && DISABLE_MD_LINK_CHECK )) && return 0
+  # Get changed markdown files (ignore /vendor)
   local mdfiles="$(echo "${CHANGED_FILES}" | grep \.md$ | grep -v ^vendor/)"
-  if [[ -n "${mdfiles}" ]]; then
+  [[ -z "${mdfiles}" ]] && return 0
+  local failed=0
+  if (( ! DISABLE_MD_LINTING )); then
     subheader "Linting the markdown files"
     lint_markdown ${mdfiles} || failed=1
+  fi
+  if (( ! DISABLE_MD_LINK_CHECK )); then
     subheader "Checking links in the markdown files"
     check_links_in_markdown ${mdfiles} || failed=1
   fi
+  return ${failed}
+}
+
+# Default build test runner that:
+# * check markdown files
+# * `go build` on the entire repo
+# * run `/hack/verify-codegen.sh` (if it exists)
+# * check licenses in `/cmd` (if it exists)
```

Comment:
this line is not accurate any more. not part of this PR, so this can be done in a separate PR

Reply:
Technically this is a nit on an outdated comment.

Reply:
Updated here and in the README.